src/main/java/com/vinylteam/vinyl/dao/jdbc/JdbcUniqueVinylDao.java

```diff
@@ -32,14 +32,14 @@ public JdbcUniqueVinylDao(HikariDataSource dataSource) {
     }
 
     @Override
-    public UniqueVinyl updateOneUniqueVinylAsHavingNoOffer(UniqueVinyl vinyl) {
+    public UniqueVinyl updateHasOffersFalse(UniqueVinyl vinyl) {
         if (vinyl.getHasOffers()){
             return vinyl;
         }
         try (Connection connection = dataSource.getConnection();
-             PreparedStatement upsertUniqueVinyl = connection.prepareStatement(UPDATE_UNIQUE_VINYL_FALSE)) {
-            setVinylParameters(upsertUniqueVinyl, vinyl);
-            int rows = upsertUniqueVinyl.executeUpdate();
+             PreparedStatement updateStatement = connection.prepareStatement(UPDATE_UNIQUE_VINYL_FALSE)) {
```

Comment:
Please rename UPDATE_UNIQUE_VINYL_FALSE to: UPDATE_UNIQUE_VINYL_SET_HAS_OFFERS_FALSE

Reply:
done